airbyte-cdk/bulk/toolkits/load-csv/src/main/kotlin/io/airbyte/cdk/load/file/csv/CSVWriter.kt

```diff
@@ -6,9 +6,13 @@ package io.airbyte.cdk.load.file.csv
 
 import io.airbyte.cdk.load.data.ObjectType
 import io.airbyte.cdk.load.data.csv.toCsvHeader
-import java.io.Writer
+import java.io.OutputStream
 import org.apache.commons.csv.CSVFormat
 import org.apache.commons.csv.CSVPrinter
 
-fun ObjectType.toCsvPrinterWithHeader(writer: Writer): CSVPrinter =
-    CSVFormat.Builder.create().setHeader(*toCsvHeader()).setAutoFlush(true).build().print(writer)
+fun ObjectType.toCsvPrinterWithHeader(outputStream: OutputStream): CSVPrinter =
+    CSVFormat.Builder.create()
+        .setHeader(*toCsvHeader())
+        .setAutoFlush(true)
+        .build()
+        .print(outputStream.writer(charset = Charsets.UTF_8))
```

Comment:
curious: was the default charset breaking something?

Reply:
https://github.com/airbytehq/airbyte/pull/46954/files#diff-d2107d506de1477d1d0df842ecf42a54dda1290922cb348991816aa14598222aR420 went in last week

(though I think we're almost at the point where you can copy the old DATs to s2-v2? which would give us equivalent coverage with the old connector, plus the extra stuff in the new test suite)